modules/mobile/app/models/mobile/v0/appointment.rb

```diff
@@ -22,11 +22,12 @@ class Appointment < Common::Resource
         'VA_VIDEO_CONNECT_GFE',
         'VA_VIDEO_CONNECT_HOME'
       )
-      STATUS_TYPE = Types::String.enum('BOOKED', 'CANCELLED', 'HIDDEN')
+      STATUS_TYPE = Types::String.enum('BOOKED', 'CANCELLED', 'HIDDEN', 'SUBMITTED')
```

Comment:
Leaving this status as "submitted" to avoid yet another translation from api to api. It doesn't make a lot of sense by itself, but I think it makes sense when paired with is_pending.